src/app/zap-templates/zcl/data-model/chip/valve-configuration-and-control-cluster.xml

```diff
@@ -19,21 +19,30 @@ limitations under the License.
 
   <bitmap name="Feature" type="bitmap32">
     <cluster code="0x0081"/>
-    <field name="TimeSync" mask="0x1"/>
-    <field name="Level" mask="0x2"/>
+    <field name="TimeSync" mask="0x01"/>
+    <field name="Level" mask="0x02"/>
   </bitmap>
 
   <bitmap name="ValveFaultBitmap" type="bitmap16">
     <cluster code="0x0081"/>
-    <field name="GeneralFault" mask="0x1"/>
-    <field name="Blocked" mask="0x2"/>
-    <field name="Leaking" mask="0x4"/>
+    <field name="GeneralFault" mask="0x01"/>
+    <field name="Blocked" mask="0x02"/>
+    <field name="Leaking" mask="0x04"/>
+    <field name="NotConnected" mask="0x08"/>
+    <field name="ShortCircuit" mask="0x10"/>
+    <field name="CurrentExceeded" mask="0x20"/>
   </bitmap>
 
   <enum name="ValveStateEnum" type="enum8">
     <cluster code="0x0081"/>
-    <item name="Open" value="0x0"/>
-    <item name="Closed" value="0x1"/>
+    <item name="Closed" value="0x0"/>
+    <item name="Open" value="0x1"/>
+    <item name="Transitioning" value="0x2"/>
+  </enum>
+
+  <enum name="StatusCodeEnum" type="enum8">
+    <cluster code="0x0081"/>
+     <item name="FailureDueToFault" value="0x02"/>
```

Comment:
Spec seems to have FAILURE_DUE_TO_FAULT?

Reply:
Do you mean this misses the underscores? I thought this is the format for enums and such types?

Reply:
I'm resolving this but we can continue the discussion.

Reply:
Let me try to check other clusters. This is a bit different from cluster to cluster 🫤

Reply:
From what I can see, most other cluster uses PascalCase for the status code names, where Disco Ball and IM uses the UpperCase with "_". 

With that said, I could also see that the IM status codes are "translated" into PascalCase as well, e.g. CONSTRAINT_ERROR becomes ConstraintError. So not really sure what is the best path forward here. @bzbarsky-apple any suggestions?

Reply:
@ReneJosefsen @fessehaeve We are going to need to generate the XML from the spec.  They should match.  In this case, I suggest a spec issue to fix the spec.

Reply:
Not a problem, I can easily adjust this on the spec side. I will get a PR up for this asap.